pkg/resource/logging-config/alloy-logging-config.go

```diff
@@ -24,7 +24,7 @@ var (
 
 func init() {
 	alloyLoggingTemplate = template.Must(template.New("logging.alloy").Funcs(sprig.FuncMap()).Parse(alloyLogging))
-	alloyLoggingConfigTemplate = template.Must(template.New("logging-config.yaml").Funcs(sprig.FuncMap()).Parse(alloyLoggingConfig))
+	alloyLoggingConfigTemplate = template.Must(template.New("logging-config.alloy").Funcs(sprig.FuncMap()).Parse(alloyLoggingConfig))
```

Comment:
Why the change of suffix? Does it have an impact on how alloy reads the file, or is it just a convention to name alloy config files `.alloy`?

Reply:
Can we then also rename the template file to align with this, also having the .yaml in the end would be more meaningful as this is a yaml file.

```suggestion
	alloyLoggingConfigTemplate = template.Must(template.New("logging-config.alloy.yaml").Funcs(sprig.FuncMap()).Parse(alloyLoggingConfig))
```

Reply:
Also, the file is already in a folder named alloy so naming the file alloy/logging-config.alloy.yaml is weird right? Unless we put all templates (promtail and alloy) in the same folder

Reply:
It makes it easier to go from the go code to the template file if they both have the same name.

Reply:
It's fixed in the new commit :)